### PR TITLE
Use proper endpoint to determine users tumonline ids

### PR DESCRIPTION
**Pull request by kordianbruck**
## Issue

This fixes the following issue(s): #552 

## Why this is useful for all students

Works for everyone - yay.

## TODO:

- [ ] Let user select which role he currently has
- [ ] Use the according ID as the main id

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/activities/wizard/WizNavCheckTokenActivity.java [hunk 1]
```diff
@@ -87,22 +88,26 @@ protected Integer onLoadInBackground(Void... arg) {
...
+                    .toString());
 
             // Save the TUMOnline id to preferences
-            String pID = getUserPIdentNr(id.get()
-                                           .toString());
-            if (pID != null) {
-                Utils.setSetting(this, Const.TUMO_PIDENT_NR, pID);
-            }
+            Utils.setSetting(this, Const.TUMO_PIDENT_NR, identity.getObfuscatd_id());
+            Utils.setSetting(this, Const.TUMO_ST_NR, identity.getObfuscatd_ids().getStudierende());
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/activities/wizard/WizNavCheckTokenActivity.java]: Could we please not use those cryptic TUMonline identifiers, but readable names?

File: app/src/main/java/de/tum/in/tumcampusapp/activities/wizard/WizNavCheckTokenActivity.java [hunk 2]
```diff
@@ -132,35 +137,4 @@ protected void onStart() {
...
-
-    /**
-     * Get the user's pident nr to identify him a little bit more. This allows information equal to
-     * the PersonDetailsActivity
-     *
-     * @param name The users full name
-     * @return the users pID, or null
-     */
-    @Nullable
-    private String getUserPIdentNr(String name) {
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/activities/wizard/WizNavCheckTokenActivity.java]: good riddance!

**Comment by kordianbruck** [comment]
We still need the user to be able to select his profile. Student, Employee or Alumni and based on that select the correct ID. The currently proposed ID is randomly chosen from the three. In the future this should be the currently main / active ID, but this is not yet implemented as it was kind of complicated. 

We could couple this with the Cafeteria Pricing that also allows to chose which role one has.

**Comment by pfent** [comment]
Meh. Since we advertise our app as "for and from students", I'd just default to the student ID. It's just for the picture anyhow....

Also: When you're not finished with a pull request, please use the "Work in progress" label 🐤

**Pull request merged**